terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/pagination/searchPathTmplAndQueryTmplAndCriteriaQuery.jsp

```diff
@@ -2,56 +2,57 @@
 <h1 class="center-content">Person Search Form</h1>
 
 <form class="center-content">
-    <spring:nestedPath path="personSearchForm">
-        <form:input path="name" />
-        <form:button id="searchButton">Search</form:button>
-    </spring:nestedPath>
+	<spring:nestedPath path="personSearchForm">

```

Comment:
This change has not canceled yet. Please remove.
commit 6b164f44bb7d2bedb3a5e2051bda109fbb68501d is included in this PR.